pilot/docker/Dockerfile.proxyv2

```diff
@@ -26,12 +27,13 @@ COPY --from=default /usr/lib/x86_64-linux-gnu/xtables/ /usr/lib/x86_64-linux-gnu
 COPY --from=default /usr/lib/x86_64-linux-gnu/ /usr/lib/x86_64-linux-gnu
 COPY --from=default /etc/iproute2 /etc/iproute2
 
-COPY istio-iptables /usr/local/bin/istio-iptables
-
 # Copy Envoy bootstrap templates used by pilot-agent
-COPY envoy_bootstrap_v2.json /var/lib/istio/envoy/envoy_bootstrap_tmpl.json
-COPY envoy_bootstrap_drain.json /var/lib/istio/envoy/envoy_bootstrap_drain.json
-COPY gcp_envoy_bootstrap.json /var/lib/istio/envoy/gcp_envoy_bootstrap_tmpl.json
+COPY --from=default /var/lib/istio/envoy/envoy_bootstrap_tmpl.json /var/lib/istio/envoy/envoy_bootstrap_tmpl.json
+COPY --from=default /var/lib/istio/envoy/envoy_bootstrap_drain.json /var/lib/istio/envoy/envoy_bootstrap_drain.json
+COPY --from=default /var/lib/istio/envoy/gcp_envoy_bootstrap_tmpl.json /var/lib/istio/envoy/gcp_envoy_bootstrap_tmpl.json
+
+COPY istio-iptables /usr/local/bin/istio-iptables
```

Comment:
While I changed the default container back to using the istio-iptables.sh shell script, the distroless container was already running the istio-iptables binary (as the script requires bash). This means that distroless *will* be running the modified istio-iptables binary with the validation code, even in non-CNI environments.

I'm not sure there is a way around that. Let me know if that change is still too big a difference for a patch release.

Reply:
in both distroless and Ubuntu, the only change here is there is an additional golang validation code that is run - nothing else changes. Is that correct? if so I don't think there's any concern